src/App.tsx

```diff
@@ -66,18 +53,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loaded && !hasError && items.length === 0 && (
                   <div className="notification is-warning" data-cy="NoPostsYet">
                     No posts yet
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length > 0 && (
-                  <PostsList
-                    posts={posts}
-                    selectedPostId={selectedPost?.id}
-                    onPostSelected={setSelectedPost}
-                  />
+                {author && loaded && !hasError && items.length > 0 && (
+                  <PostsList posts={items} selectedPostId={selectedPost?.id} />
```

Comment:
Do you use `selectedPost` in App.tsx only to pass `id` as prop? In this case, you can just get it from Redux Selector in `PostList` and remove here

Reply:
No, I don't only use id